comps/comps-foreman-el8.xml

```diff
@@ -187,60 +338,201 @@
+      <packagereq type="default">rubygem-rubyipmi-doc</packagereq>
+      <packagereq type="default">rubygem-ruby-libvirt-doc</packagereq>
+      <packagereq type="default">rubygem-ruby_parser-doc</packagereq>
+      <packagereq type="default">rubygem-safemode-doc</packagereq>
+      <packagereq type="default">rubygem-sassc-doc</packagereq>
+      <packagereq type="default">rubygem-sassc-rails-doc</packagereq>
       <packagereq type="default">rubygem-sass-doc</packagereq>
       <packagereq type="default">rubygem-sass-rails-doc</packagereq>
+      <packagereq type="default">rubygem-scoped_search-doc</packagereq>
+      <packagereq type="default">rubygem-secure_headers-doc</packagereq>
+      <packagereq type="default">rubygem-sequel-doc</packagereq>
+      <packagereq type="default">rubygem-sexp_processor-doc</packagereq>
+      <packagereq type="default">rubygem-sidekiq-doc</packagereq>
+      <packagereq type="default">rubygem-signet-doc</packagereq>
       <packagereq type="default">rubygem-sinatra-doc</packagereq>
+      <packagereq type="default">rubygem-spice-html5-rails-doc</packagereq>
       <packagereq type="default">rubygem-sprockets-doc</packagereq>
+      <packagereq type="default">rubygem-sprockets-rails-doc</packagereq>
       <packagereq type="default">rubygem-sqlite3-doc</packagereq>
+      <packagereq type="default">rubygem-sshkey-doc</packagereq>
+      <packagereq type="default">rubygem-statsd-instrument-doc</packagereq>
+      <packagereq type="default">rubygem-text-doc</packagereq>
       <packagereq type="default">rubygem-thor-doc</packagereq>
       <packagereq type="default">rubygem-thread_safe-doc</packagereq>
       <packagereq type="default">rubygem-tilt-doc</packagereq>
       <packagereq type="default">rubygem-tzinfo-doc</packagereq>
+      <packagereq type="default">rubygem-uber-doc</packagereq>
       <packagereq type="default">rubygem-uglifier-doc</packagereq>
+      <packagereq type="default">rubygem-unf-doc</packagereq>
+      <packagereq type="default">rubygem-unf_ext-doc</packagereq>
+      <packagereq type="default">rubygem-unicode-display_width-doc</packagereq>
+      <packagereq type="default">rubygem-unicode-doc</packagereq>
+      <packagereq type="default">rubygem-useragent-doc</packagereq>
+      <packagereq type="default">rubygem-validates_lengths_from_database-doc</packagereq>
+      <packagereq type="default">rubygem-webpack-rails-doc</packagereq>
       <packagereq type="default">rubygem-websocket-driver-doc</packagereq>
       <packagereq type="default">rubygem-websocket-extensions-doc</packagereq>
+      <packagereq type="default">rubygem-will_paginate-doc</packagereq>
+      <packagereq type="default">rubygem-wirb-doc</packagereq>
+      <packagereq type="default">rubygem-x-editable-rails-doc</packagereq>
+      <packagereq type="default">rubygem-xmlrpc-doc</packagereq>
+      <packagereq type="default">tfm-rubygem-m-doc</packagereq>
```

Comment:
This looks odd on EL8.

Reply:
This was an artifact from using the add_gem_package script to sort for me. It's been removed.